src/Mod/Sketcher/Gui/CommandConstraints.cpp

```diff
@@ -2123,6 +2140,92 @@ class DrawSketchHandlerDimension : public DrawSketchHandler
                     ActDist = bigradius - smallradius - intercenterdistance;
                 }
 
+                Gui::cmdAppObjectArgs(Obj, "addConstraint(Sketcher.Constraint('Distance',%d,%d,%f))",
+                    GeoId1, GeoId2, ActDist);
+            }
+            else if (isArcOfCircle(*geo1) && isArcOfCircle(*geo2)) { // ArcOfCircle - ArcOfCircle case
```

Comment:
Same here, you shouldn't copy paste whole blocks unecessarily. Shrink these to one block : 

`            else if ((isCircle(*geo1) || isArcOfCircle(*geo1)) && (isCircle(*geo2) || isArcOfCircle(*geo2)))`